### PR TITLE
update cmake of traffic signal messages

### PR DESCRIPTION
**Pull request by takayuki5168**
update cmake of traffic signal messages for https://github.com/tier4/autoware_auto_msgs/pull/21

### CODE REVIEW

**Comment by takayuki5168** [comment]
@taichiH We forgot to update CMakeLists.txt, sorry

**Review by taichiH — APPROVED** [review]
sorry... I forgot to commit CMakeLists.txt.

File: autoware_auto_perception_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -29,9 +29,12 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   "msg/TrackedObjects.idl"
   "msg/TrafficLight.idl"
   "msg/TrafficSignal.idl"
+  "msg/TrafficSignalStamped.idl"
+  "msg/TrafficSignalWithJudge.idl"
   "msg/TrafficSignalArray.idl"
   "msg/TrafficLightRoi.idl"
   "msg/TrafficLightRoiArray.idl"
+  "msg/LookingTrafficSignal.idl"
```
Comment (taichiH) [inline autoware_auto_perception_msgs/CMakeLists.txt]: please sort alphabetically

**Review by taichiH — CHANGES_REQUESTED** [review]

**Comment by takayuki5168** [comment]
Once build failed. I don't know why but I jsut reopened this PR
https://github.com/tier4/autoware_auto_msgs/runs/4179826814?check_suite_focus=true

File: autoware_auto_perception_msgs/CMakeLists.txt (hunk 1 above)
Comment (takayuki5168) [reply autoware_auto_perception_msgs/CMakeLists.txt]: https://github.com/tier4/autoware_auto_msgs/pull/22/commits/8af155a4667d3b585898b80693a9229f72e78220

**Comment by takayuki5168** [comment]
image
Still build failed...

**Comment by takayuki5168** [comment]
It seems final is pre-defined word in C++, and fail formatting? :thinking:

**Comment by takayuki5168** [comment]
cc @kenji-miyake

**Comment by taichiH** [comment]
@takayuki5168 @satoshi-ota 
`final` is a reserved word, so when I changed it, the test passed on my local environment.
How about changing to other word? (result?)

https://github.com/tier4/autoware_auto_msgs/blob/0f87daa6f06d0c0513a502d0e192be113181692a/autoware_auto_perception_msgs/msg/LookingTrafficSignal.idl#L11

**Comment by takayuki5168** [comment]
@tkimura4 ↑

**Comment by takayuki5168** [comment]
@taichiH Everyone accepts the idea to renaming to `result`. Could you approve again and merge.

**Review by taichiH — APPROVED** [review]

**Pull request merged**